app/src/main/java/ca/cgagnier/wlednativeandroid/ui/homeScreen/list/DeviceListItem.kt

```diff
@@ -272,6 +272,18 @@ fun DeviceInfoTwoRows(
                         .height(20.dp)
                 )
             }
+
+            if(device.hasBattery){
```

Comment:
Nit: You're missing a space between the if and the condition. I recommend running the default code reformat on the code you changed to try to have a consistant style.